prefect_azure/credentials.py

```diff
@@ -284,10 +293,9 @@ def example_get_container_client_flow():
         return container_client
 
 
-@dataclass
-class MlAzureCredentials:
+class MlAzureCredentials(Block):
```

Comment:
Update to match block type name?
```suggestion
class AzureMlCredentials(Block):
```